extendr-api/src/graphics/device_descriptor.rs

```diff
@@ -0,0 +1,241 @@
+use super::{color::Color, FontFace, LineType};
+
+// R internals says:
+//
+// There should be a ‘pointsize’ argument which defaults to 12, and it should
+// give the pointsize in big points (1/72 inch). How exactly this is interpreted
+// is font-specific, but it should use a font which works with lines packed 1/6
+// inch apart, and looks good with lines 1/5 inch apart (that is with 2pt
+// leading).
+const POINTSIZE: f64 = 12.0;
+
+const PT: f64 = 1.0 / 72.0;
+const PT_PER_INCH: f64 = 72.0;
+
+// R internals says:
+//
+// where ‘fnsize’ is the ‘size’ of the standard font (cex=1) on the device, in
+// device units.
+//
+// and it seems the Postscript device chooses `pointsize` as this.
+const FONTSIZE: f64 = POINTSIZE;
+
+// R internal says:
+//
+// The default size of a device should be 7 inches square.
+const WIDTH_INCH: f64 = 7.0;
+const HEIGH_INCH: f64 = 7.0;
+
+#[allow(dead_code)]
+pub(crate) enum CanHAdjOption {
+    NotSupported = 0,
+    FixedAdjustment = 1,
+    VariableAdjustment = 2,
+}
+
+pub enum GraphicDeviceCapabilityTransparency {
```

Comment:
What about using abbreviations, e.g. `GDCTransparency`?

Reply:
I agree this is a bit too long, but I think `GDC` a bit too short. I need to invent a nicer name...